src/components/my-history/custom-my-history.vue

```diff
@@ -0,0 +1,140 @@
+          {{ $t('myHistory.subtitle') }}
+        </h2>
+        <div
+          v-html="$t('myHistory.message')"
+          class="x-list x-list--gap-05 x-title3 x-font-color--neutral-35"
+        ></div>
+      </div>
+      <div class="x-list x-list__item--expand x-border-color--neutral-95 x-border-width--top-01">
+        <MyHistory v-if="isMyHistoryEnabled && $x.fullHistoryQueries.length">
+          <template #date="{ date }">
+            <div
+              class="
+                x-padding--top-05 x-padding--bottom-05
+                x-font-color--neutral-35
+                x-font-weight--bold
+              "
+              :class="$x.device === 'desktop' ? 'x-padding--07' : 'x-padding--05'"
+            >
+              {{ date }}
+            </div>
+          </template>
+          <template #suggestion-content="{ suggestion, formatTime }">
+            <div
+              class="
+                x-list x-list--horizontal x-list--gap-05
+                x-padding--bottom-04 x-padding--top-04
+              "
+            >
+              <HistoryIcon :class="{ 'x-icon--l': $x.device === 'mobile' }" />
+              <div class="x-list x-list--gap-01">
+                <span>{{ suggestion.query }}</span>
+                <span class="x-small x-font-color--neutral-35">
+                  {{ formatTime(suggestion.timestamp) }}
+                </span>
+              </div>
+            </div>
+          </template>
+          <template #suggestion-remove-content>
+            <CrossTinyIcon :class="{ 'x-icon--l': $x.device === 'mobile' }" />
+          </template>
+        </MyHistory>
+        <div v-else class="x-list x-list--align-center x-list__item--expand">
+          <NoHistoryIcon
+            class="x-flex-no-shrink"
+            :class="{ 'x-no-history-icon--bw': !isMyHistoryEnabled }"
+          />
+          <span
+            class="
+              x-title3
+              x-padding--05
+              x-font-weight--bold
+              x-font-color--neutral-35
+              x-list x-list--justify-center x-list__item--expand
+            "
+          >
+            {{ $t('myHistory.noHistory') }}
+          </span>
+        </div>
+      </div>
+    </div>
+  </div>
+</template>
+
+<script lang="ts">
+  import { CrossTinyIcon, HistoryIcon } from '@empathyco/x-components';
+  import { MyHistory } from '@empathyco/x-components/history-queries';
+  import { Component, Prop, Vue } from 'vue-property-decorator';
+  import MyHistoryIcon from './my-history-icon.vue';
+  import MyHistoryIconBw from './my-history-icon-bw.vue';
+  import NoHistoryIcon from './no-history-icon.vue';
+
+  @Component({
+    components: {
+      CrossTinyIcon,
+      HistoryIcon,
+      MyHistory,
+      MyHistoryIcon,
+      MyHistoryIconBw,
+      NoHistoryIcon
+    }
+  })
+  export default class CustomMyHistory extends Vue {
+    @Prop({ default: true })
+    public isMyHistoryEnabled!: boolean;
```

Comment:
Can we animate the list of history queries?

```typescript
    protected animation = StaggeredFadeAndSlide;
```

Reply:
Sure, good point! I've added the animation